website/docs/r/muting_schedule.html.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: 'sumologic'
+page_title: 'SumoLogic: sumologic_muting_schedule'
+description: |-
+  Provides the ability to create, read, delete, and update muting schedule.
+---
+
+# sumologic_muting_schedule
+
+Provides the ability to create, read, delete, and update [MutingSchedule][1].
+
+## Example One Time Muting Schedule For All monitor
```

Comment:
nit: `One-time Muting Schedule at ... for All Monitors`

Reply:
updated.